Fw/Python/src/fprime/fbuild/cmake.py

```diff
@@ -461,7 +460,10 @@ def _run_cmake(
         :return: (stdout, stderr)
         Note: !!! this function has potential File System side-effects !!!
         """
-        cm_environ = copy.copy(os.environ)
+        if environment is None:
```

Comment:
Ugh.  I hate this fact about python.  It is the correct fix.

Reply:
Sadly yes. Having a dictionary as the default value without a 'is none' can cause some weird errors